observer/test/platform-stats.test.js

```diff
@@ -17,10 +17,18 @@ describe('platform-stats-generator', () => {
   let today
   beforeEach(async () => {
     await pgClient.query('DELETE FROM daily_reward_transfers')
-
+    await pgClient.query('DELETE FROM participants')
     // Run all tests inside a transaction to ensure `now()` always returns the same value
     // See https://dba.stackexchange.com/a/63549/125312
     // This avoids subtle race conditions when the tests are executed around midnight.
+
+    // Insert participants for testing (addresses mapped to IDs)
+    await pgClient.query(`
+      INSERT INTO participants (id, participant_address)
+      VALUES (1, 'address1'), (2, 'address2')
+    `)
```

Comment:
FIXME: use `mapParticipantsToIds` and save the ids generated by the database instead of using hard-coded ID values `1` and `2`.